lib/activity.php

```diff
@@ -130,24 +128,33 @@ public function translate($app, $text, $params, $stripPath, $highlightParams, $l
 	 * @return array|false
 	 */
 	public function getSpecialParameterList($app, $text) {
-		/*if ($app === 'files_sharing') {
+		if ($app === 'passman') {
 			switch ($text) {
-				case self::SUBJECT_REMOTE_SHARE_ACCEPTED:
-				case self::SUBJECT_REMOTE_SHARE_DECLINED:
-				case self::SUBJECT_REMOTE_SHARE_UNSHARED:
+				case self::SUBJECT_ITEM_CREATED:
+				case self::SUBJECT_ITEM_EDITED:
+				case self::SUBJECT_ITEM_DELETED:
+				case self::SUBJECT_ITEM_RECOVERED:
+				case self::SUBJECT_ITEM_DESTROYED:
+					return array(
+						0 => 'passman',
+						1 => 'username',
```

Comment:
this adds the avatar to the username,
passman is not used atm, but could be used at some point to link the passman parameter to the password page